AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginLazyLoadTests/LL12/CompositePK/CompositePKChild+Schema.swift

```diff
@@ -66,4 +66,4 @@ extension ModelPath where ModelType == CompositePKChild {
   public var updatedAt: FieldPath<Temporal.DateTime>   {
       datetime("updatedAt") 
     }
-}
```

Comment:
I think this PR is missing some changes after the rebase to `AWSDataStoreLazyLoadCompositePKTests`, such as changing 
```
assertLazyReference(queriedCompositePKChild._parent,
                            state: .notLoaded(identifiers: [.init(name: "@@primaryKey", value: savedParent.identifier)]))
```
to
```
assertLazyReference(queriedCompositePKChild._parent,
                            state: .notLoaded(identifiers: [.init(name: "CompositePKParent.keys.customId", value: savedParent.customId),
.init(name: "CompositePKParent.keys.content", value: savedParent.content]))
```

etc..